example-adapter/README.md

```diff
@@ -1,144 +1,153 @@
-
-Clone the repository to get access to the sources:
+The fastest and easiest way to try out the application is by using the Spring Boot Maven plugin.
+To do this, you need to have a working installation of [Maven](https://maven.apache.org/).
 
 ```
-git clone https://github.com/nordic-institute/xrd4j.git
+mvn spring-boot:run
 ```
-
-The example adapter project root is under the `example-adapter` directory.
-
-Build the project from the project root with:
+After that the application is accessible at:
 
 ```
-mvn clean install
+http://localhost:8080/example-adapter/Endpoint
 ```
 
-After a successful build, at project root directory run:
+The WSDL description is accessible at:
 
 ```
-docker build -t example-adapter .
+http://localhost:8080/example-adapter/Endpoint?wsdl
 ```
 
-This will create a new docker image named `example-adapter`. A container based on the image can be run with:
+## Software Requirements
 
-```
-docker run -p 8080:8080 example-adapter
-```
+* Java 8 or Java 11
+* Maven 3.x
+* Docker (*optional*)
 
-This will start the container and listen to port `8080`. On Linux, the container should be available at `localhost`. On Windows use `docker-machine ip` command to get Docker host’s IP address.
+## Development Environment
 
-The example service can now be found at `http://localhost:8080/example-adapter-x.x.x-SNAPSHOT/Endpoint` where `x.x.x` needs to be replaced by the version that can be verified from the maven project file, or the archives produced by the build.
-Example requests can be found in the `examples` directory.
+Setting up an environment for example-related development is explained [here](Setting-up-Development-Environment.md).
 
-See [Usage section](#usage) for further instructions.
+## Installation
 
-### Software Requirements
+X-Road Example Adapter can be installed and run in the following ways:
 
-* Java 8
-* Tomcat 6 or later
+* Deploying `example-adapter-x.x.x.war` into a web container such as Tomcat.
+* Using Docker to run X-Road Example Adapter.
 
-### Development Environment
+### Web container
 
-Setting up an environment for example-related development is explained [here](Setting-up-Development-Environment.md).
+Build X-Road Example Adapter and deploy it to a Java application server, e.g., Tomcat.
 
-### Installation
+* Build the project and produce ```example-adapter-x.x.x.war``` file.
+* Copy the file ```tomcat.home/webapps``` folder.
+* Start/restart Tomcat.
+* 
+### Docker
 
-**N.B.** If you intend to connect the services to an X-Road Security Server you must update your server's IP address / host name to the WSDL file. Look for the below line and replace the default URL with your server's IP / host name.
+You can create a Docker image to run X-Road Example Adapter inside a container, using the provided Dockerfile.
+Before building the image, you must build the war file.
 
-```XML
-<soap:address location="http://localhost:8080/example-adapter-x.x.x-SNAPSHOT/Endpoint" />
+```
+mvn clean install
```

Comment:
I think this could be `mvn clean verify package` instead. I believe that `install` also installs the package to the local maven repository, which probably is not required in this case. `package` should just create the jar/war files and leave it at that. `verify` is needed because it appears that just `package` does not run checks, e.g. the dependency checker.

Reply:
Changed this to `mvn clean verify` since installing the package to local maven repository isn't required in this case. However, `verify` already includes `package` so there's no need to include `package` explicitly.